media/css/custom.scss

```diff
@@ -471,3 +476,34 @@ summary {
 
 @import "faculty_structure";
 @import "honours_board";
+
+.relax-item {
+  width: 200px;
+  height: 200px;
+  background: lavenderblush;
+  float: left;
+  margin-left: 20px;
+  margin-top: 10px;
+}
+
+.relax-img {
+  width: 100%;
+  height: 150px;
+  background: silver;
```

Comment:
Color `silver` should be written in hexadecimal form as `#c0c0c0`<br>Color literals like `silver` should only be used in variable declarations; they should be referred to via variable everywhere else.